merging-configmaps-gitops/README.md

```diff
@@ -0,0 +1,84 @@
+# Merging config in a gitops context
+
+Our app platform utilizes configmaps / secrets heavily to pass configuration into apps.
+
+*The following RFC will focus on configmaps but the same problem and solution are also applicable for secrets!*
+
+These configmaps are often either shared between multiple instances of the same app or have shared values between apps.
+In gitops the [intended solution](https://kubectl.docs.kubernetes.io/guides/introduction/kustomize/#2-create-variants-using-overlays) for both of these cases is to use `bases` and `overwrites` which is achieved through `kustomize`.
```

Comment:
It looks like `kustomize` calls this feature `overlays` instead of `overwrites`.

Reply:
Fixed :+1: